src/Symfony/Component/Security/Acl/Permission/PermissionMapInterface.php

```diff
@@ -39,4 +39,11 @@ public function getMasks($permission, $object);
      * @return bool
      */
     public function contains($permission);
+
+    /**
+     * Returns a new instance of the MaskBuilder used in the permissionMap
+     *
+     * @return MaskBuilderInterface
+     */
+    public function getMaskBuilder();
```

Comment:
Unfortunately, this is a BC break, we cannot change an interface in a minor version. Two options here: don't add the method to the interface and add it only in the master branch (for 3.0) or create another interface.